kepler/base/scc.yaml

```diff
@@ -0,0 +1,26 @@
+kind: SecurityContextConstraints
```

Comment:
can you move this to the [cluster-scope](https://github.com/operate-first/apps/tree/master/cluster-scope/base/security.openshift.io/securitycontextconstraints) folder, as this is where we manage all our privileged & cluster scoped resources (in general anything a project admin cannot deploy).